modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/loader/parser/java/JavaExtensionModelParserTestCase.java

```diff
@@ -234,4 +278,37 @@ private static class ExtensionAnnotatedWithHighMMV {
     String extensionParameter;
   }
 
+  private static class SimplePojoWithTime {
+
+    @Parameter
+    private String name;
+
+    @Parameter
+    private Integer age;
+
+    @Parameter
+    private LocalDateTime dateTime;
+
+    public SimplePojoWithTime(String name, Integer age, LocalDateTime dateTime) {
```

Comment:
Needed as build failed with the following
Extension 'MixedConfigurationsAnnotationExtension' has definition errors:
ERRORS:
	* Type 'SimplePojoWithTime' must override equals and hashCode
	* Type 'SimplePojoWithTime' does not have a default constructor

Reply:
default doesn't mean with parameters